fontc/src/change_detector.rs

```diff
@@ -249,28 +289,37 @@ impl ChangeDetector {
 
     pub fn mark_be_change(&self) -> bool {
         // Glyphs produce anchors and we need anchors
-        !self.glyphs_changed().is_empty()
+        !self.glyphs_changed.is_empty()
             || !self
                 .be_paths
                 .target_file(&BeWorkIdentifier::Marks)
                 .is_file()
     }
 
     pub fn kerning_be_change(&self) -> bool {
-        self.kerning_ir_change()
+        self.kerning_groups_ir_change()
             || !self
                 .be_paths
                 .target_file(&BeWorkIdentifier::Kerning)
                 .is_file()
     }
 
-    pub fn kerning_ir_change(&self) -> bool {
+    pub fn kerning_groups_ir_change(&self) -> bool {
         self.static_metadata_ir_change()
             || self.glyph_order_ir_change()
             || self.current_inputs.features != self.prev_inputs.features
             || !self
                 .ir_paths
-                .target_file(&FeWorkIdentifier::Kerning)
+                .target_file(&FeWorkIdentifier::KerningGroups)
+                .is_file()
+    }
+
+    pub fn kerning_at_ir_change(&self, at: NormalizedLocation) -> bool {
```

Comment:
In a world where we had a more concrete graph representation ideally we would be to use that to determine what gets invalidated, and then we could simplify this API to something like,

```rust
fn is_up_to_date(&self, work_id: &WorkId) -> bool { ..}
```

or something?

Reply:
+1 but not in this PR :D